tests/verilog-conversion/test_hdl.txt

```diff
@@ -0,0 +1,212 @@
+  [

```

Comment:
Please do a pull and re-run. In the newest version, the <null child> no longer appears.